Source/GUI/Settings/SConfig.h

```diff
@@ -64,6 +64,11 @@ class SConfig
 
   bool ownsSettingsFile() const;
 
+  bool getAutoloadLastFile() const;
+  void setAutoloadLastFile(bool enabled);
+  QString getLastLoadedFile() const;
+  void setLastLoadedFile(const QString fileName);
```

Comment:
Another case where a const reference should be used:

```suggestion
  void setLastLoadedFile(const QString& fileName);
```

Reply:
Done